src/color-picker/color-picker.css

```diff
@@ -4,6 +4,13 @@
 	gap: 1em;
 }
 
+#space_picker {
+	flex: 0 1 100%; /* occupy the entire row */
+	padding: initial;
+	border: none;
```

Comment:
I'd instead make it transparent and show on hover

Reply:
Good idea! Thanks. Done.